addons/payment_ingenico/controllers/main.py

```diff
@@ -23,7 +23,7 @@ class OgoneController(http.Controller):
         '/payment/ogone/decline', '/payment/ogone/test/decline',
         '/payment/ogone/exception', '/payment/ogone/test/exception',
         '/payment/ogone/cancel', '/payment/ogone/test/cancel',
-    ], type='http', auth='public', csrf=False, method=['GET', 'POST'])
+    ], type='http', auth='public', csrf=False, methods=['GET', 'POST'])
```

Comment:
Could just remove this one entirely no?

Reply:
> Could just remove this one entirely no?

sounds good, and wrt `csrf=False`, those parameters are expected to be signed, so that takes care of CSRF